frontend/views/components/group-creation-steps/GroupName.vue

```diff
@@ -61,7 +61,10 @@ export default ({
     sbp('okTurtles.events/on', AVATAR_EDITED, this.updateGroupPictureByEditor)
   },
   mounted () {
-    this.$refs.name.focus()
+    window.setTimeout(() => {
+      this.$refs.name.focus()
+    }, 300)
+
```

Comment:
In general, we want to add code comments to code that someone might find strange, odd, confusing, or unclear. This is a great example, as someone reading this would have absolutely no idea why this delay is here, and the answer is quite unintuitive. So my only request for this PR would be to add a comment referencing the issue this is closing and explaining why this is here.